src/GloryDuckTables2019Lkl.cc

```diff
@@ -207,17 +207,17 @@ Int_t GloryDuckTables2019Lkl::ReadGloryDuckInputData(TString filename)
   else
     cout << "GloryDuckTables2019Lkl::ReadGloryDuckInputData (" << GetName() << ") Message: Reading masses and <sigmav> values from file " << filename << endl;
 
-  // get the number of lines of the file
-  Int_t numRows = 0;
+  // get the number of lines (=number of <sv> values - 1) in the file
```

Comment:
To be more clear I would write something like:

// get the number of <sv> values (which is one less than the number of lines in the file)

Reply:
Ok!